SPECS/lttng-tools/lttng-tools.spec

```diff
@@ -16,7 +16,7 @@ BuildRequires: elfutils-devel
 BuildRequires: popt-devel
 BuildRequires: userspace-rcu-devel >= 0.8.0
 BuildRequires: lttng-ust-devel >= 2.9.0
-Requires:      lttng-ust = %{version}
+Requires:      lttng-ust >= 2.13.1
 Requires:      userspace-rcu
```

Comment:
I think it would be best to have the same version constraint here as the build-time requirement on userspace-rcu-devel.